tests/subsys/pm/power_states/src/main.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+
+int main(void)
+{
+
+	while (1) {
+		printk("\nGoing to k_cpu_idle.\n");
+		k_msleep(400);
+		printk("\nWake Up.\n");
```

Comment:
can we use this tests\subsys\pm\power_mgmt_soc\src\power_mgmt.c, instead of the sleep time?

Reply:
I don't know yet, let me see.

Reply:
Yep, get the information from the device tree is the way to go imho, otherwise it will not be portable and the previous comment is useless.